lib/airplayer/youtube_dl.rb

```diff
@@ -0,0 +1,92 @@
+require 'shellwords'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.